toolkit/docs/building/building.md

```diff
@@ -523,6 +526,17 @@ sudo make hydrate-rpms PACKAGE_ARCHIVE=./rpms.tar.gz
 
 > Normal build. No hydrated RPMs will be used.
 
+#### `DELTA_BUILD=...`]
+
+##### `DELTA_BUILD=`**`y`**]
+
+> Delta build. Used for fast delta builds where published packages are pre-populated and only new or added packages are built.
+
+##### `DELTA_BUILD=`**`n`** *(default)*
```

Comment:
Default named here doesn't match the default in the Makefile

Reply:
Thanks. Fixed the makefile to set the default to n